lib/components/ms-toggle/MsBooleanToogle.vue

```diff
@@ -0,0 +1,75 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="ms-boolean-toggle">
+    <ion-text
+      type="button"
+      class="button-view button-medium"
+      :class="{ 'button-disabled': modelValue === Answer.No }"
+      :disabled="modelValue === Answer.Yes"
+      @click="$emit('update:modelValue', Answer.No)"
+    >
+      {{ leftValue }}
+    </ion-text>
+    <ion-text
+      type="button"
+      class="button-view button-medium"
+      :disabled="modelValue === Answer.No"
+      :class="{ 'button-disabled': modelValue === Answer.Yes }"
+      @click="$emit('update:modelValue', Answer.Yes)"
+    >
+      {{ rightValue }}
+    </ion-text>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { Answer } from '@lib/components/ms-toggle/types';
+
+defineProps<{
+  modelValue: Answer;
+  leftValue: string;
```

Comment:
Should also be renamed. We associate `Yes` and `No` values, but I have no way of knowing which is which: is `leftValue` equivalent to Yes ? Or No ? Plus, they're not values, they're labels.